<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>BatchProcessDialog</class>
 <widget class="QDialog" name="BatchProcessDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>501</width>
    <height>255</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Batch Processing Options</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_3">
   <item>
    <widget class="QGroupBox" name="groupBox">
     <property name="title">
      <string>Input/Output Options</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_3">
        <item>
         <widget class="QLabel" name="label_3">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Fixed" vsizetype="Preferred">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="text">
           <string>Mode:</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QRadioButton" name="currentLGModeRadioButton">
          <property name="text">
           <string>Current Settings Only</string>
          </property>
          <property name="checked">
           <bool>true</bool>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QRadioButton" name="allLightGroupsModeRadioButton">
          <property name="text">
           <string>Individual Light Groups</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout">
        <item>
         <widget class="QLabel" name="label">
          <property name="text">
           <string>Input Directory:</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="inputDirectoryLineEdit"/>
        </item>
        <item>
         <widget class="QToolButton" name="browseForInputDirectoryButton">
          <property name="text">
           <string>...</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_2">
        <item>
         <widget class="QLabel" name="label_2">
          <property name="text">
           <string>Output Directory:</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="outputDirectoryLineEdit"/>
        </item>
        <item>
         <widget class="QToolButton" name="browseForOutputDirectoryButton">
          <property name="text">
           <string>...</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="groupBox_2">
     <property name="title">
      <string>Format Options</string>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout_4">
      <item>
       <widget class="QCheckBox" name="tonemapCheckBox">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="text">
         <string>Apply Tonemapping:</string>
        </property>
        <property name="checked">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QComboBox" name="imageFormatComboBox">
        <item>
         <property name="text">
          <string>PNG</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>JPEG</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>BMP</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>TIFF</string>
         </property>
        </item>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>BatchProcessDialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>248</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>BatchProcessDialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>tonemapCheckBox</sender>
   <signal>toggled(bool)</signal>
   <receiver>imageFormatComboBox</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>39</x>
     <y>190</y>
    </hint>
    <hint type="destinationlabel">
     <x>208</x>
     <y>213</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
